chain/ethereum/src/chain.rs

```diff
@@ -746,11 +748,12 @@ impl TriggersAdapterTrait<Chain> for TriggersAdapter {
         &self,
         ptr: BlockPtr,
         offset: BlockNumber,
+        root: Option<BlockHash>,
```

Comment:
What does `root` mean here? Should be explained in a comment what that is for

Reply:
Added comments